pkg/cluster/nodes/util.go

```diff
@@ -44,3 +46,23 @@ func GetControlPlaneEndpoint(allNodes []Node) (string, error) {
 
 	return fmt.Sprintf("%s:%d", loadBalancerIP, loadbalancer.ControlPlanePort), nil
 }
+
+// to avoid the DNS crash we need copy host's /etc/resolv.conf to node
+// ref: https://github.com/kubernetes-sigs/kind/pull/484#issuecomment-489469044
+func addResolve(node *Node) error {
+	resolv, err := ioutil.ReadFile("/etc/resolv.conf")
+	if err != nil {
+		return errors.Wrap(err, "failed to read /etc/resolv.conf")
+	}
+
+	// filter the loopback addresses
+	re := regexp.MustCompile("(?m)[\r\n]+^.*127.0.0.1.*$")
```

Comment:
the loopback address network is 127.0.0.0/8 so we should filter all the address in that range, ie. from 127.0.0.1 to 127.255.255.255
I think that it should be something like this 127.\d+.\d+.\d+

Reply:
:+1:  Thank you for correcting me, I missed it.